packages/faustwp-cli/src/healthCheck/validateFaustEnvVars.ts

```diff
@@ -28,4 +28,28 @@ export const validateFaustEnvVars = () => {
       'Please make sure your production Faust app runs with a WordPress instance on https!',
     );
   }
+  if (getWpSecret()) {
+    // send secret key
+    const apiUrl = `${getWpUrl()}/wp-json/faustwp/v1/validate_secret_key`;
+    const headers = {
+      'x-faustwp-secret': getWpSecret() || '',
+    };
+    try {
+      const response = await fetch(apiUrl, {
+        headers,
+        method: 'POST',
+        timeout: 30000, // 30 seconds timeout
+      } as unknown as RequestInit);
```

Comment:
No need to cast this type
```suggestion
      });
```